osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -30,33 +33,42 @@ public class OsuModRelax : ModRelax, IUpdatableByPlayfield, IApplicableToDrawabl
         private bool wasLeft;
 
         private OsuInputManager osuInputManager;
-
+        private Bindable<OsuFramedReplayInputHandler> replayInputHandler = new Bindable<OsuFramedReplayInputHandler>();
+        private Score score;
+        private readonly BindableBool replayLoaded = new BindableBool();
+        private DrawableRuleset<OsuHitObject> drawableRuleset;
         private ReplayState<OsuAction> state;
         private double lastStateChangeTime;
 
-        private bool hasReplay;
+        public void ApplyToPlayer(Player player)
+        {
+            replayInputHandler.Value = (OsuFramedReplayInputHandler)osuInputManager.ReplayInputHandler;
+            replayInputHandler.BindValueChanged(_ => updateState(), true);
```

Comment:
I don't think this `BindValueChanged` event is ever invoked (except at this initialization) because `replayInputHandler.Value` is not assigned elsewhere.
Then, the `Bindable` is not needed.